docs/internals/contributing/committing-code.txt

```diff
@@ -11,8 +11,7 @@ contribute code to Django, look at :doc:`writing-code/working-with-git` instead.
 Handling pull requests
 ======================
 
-Since Django is hosted on GitHub, patches are provided in the form of pull
-requests.
+Django is hosted on GitHub and uses pull requests (PRs) for contributions.
```

Comment:
The info was correct as is. Concept of sending a patch is a bit outdated, but the general term "patch" still has the correct and up to date meaning here.